cmd/preview/deploy.go

```diff
@@ -276,3 +289,13 @@ func getExpandedName(name string) string {
 	}
 	return expandedName
 }
+
+func getPreviewURL(ctx context.Context, name string) (string, error) {
+	oktetoURL := okteto.GetURL()
+	previewEnv, err := okteto.GetPreviewEnvByName(ctx, name, name)
+	if err != nil {
+		return "", err
+	}
+	previewURL := fmt.Sprintf("%s/#/spaces/%s?resourceId=%s", oktetoURL, name, previewEnv.ID)
```

Comment:
I think this ID refers to the space. I would just redirect to `%s/#/spaces/%s` here

Reply:
If they follow the same as in spaces this `resourceId` is different from space id. However, as only the preview environment application will be available on the space we could remove the resourceId